### PR TITLE
Update menu inventory

### LINKED ISSUE #85: inventory- update menu settings
The Service Provider, Provider Categories, Details, and Details Categories should be in the Settings submenu

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: inventory/i18n/fr/menu.inventory.left.json [hunk 1]
```diff
@@ -1,6 +1,129 @@
...
+        "item.customers-archived": {
+            "label": "Archivés",
+            "help": ""
+        },
+        "item.sale": {
+            "label": "Ventes",
+            "help": ""
+        },
+        "item.all.receivables": {
+            "label": "Receivables",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.left.json]: I used "Percevable" in french for receivable

File: inventory/i18n/fr/menu.inventory.left.json [hunk 2]
```diff
@@ -1,6 +1,129 @@
...
+        "item.receivable.pending": {
+            "label": "En attente",
+            "help": ""
+        },
+        "item.sale.entry": {
+            "label": "Entrées de vente",
+            "help": ""
+        },
+        "item.serves": {
+            "label": "Advanced",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.left.json]: Avancé

File: inventory/i18n/fr/menu.inventory.left.json [hunk 3]
```diff
@@ -1,6 +1,129 @@
...
+        "inventory.services.services": {
+            "label": "Services",
+            "help": ""
+        },
+        "item.software": {
+            "label": "Logiciels",
+            "help": ""
+        },
+        "inventory.services.subscription": {
+            "label": "Subscriptions",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.left.json]: Abonnements

File: inventory/i18n/fr/menu.inventory.left.json [hunk 4]
```diff
@@ -1,6 +1,129 @@
...
+        "item.access": {
+            "label": "Accès",
+            "help": ""
+        },
+        "item.instance": {
+            "label": "Instances",
+            "help": ""
+        },
+        "item.ips": {
+            "label": "Ip Addresses",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.left.json]: Adresses IP

File: inventory/i18n/fr/menu.inventory.left.json [hunk 5]
```diff
@@ -1,6 +1,129 @@
...
+        "item.sale": {
+            "label": "Ventes",
+            "help": ""
+        },
+        "item.all.receivables": {
+            "label": "Receivables",
+            "help": ""
+        },
+        "item.receivablesqueue": {
+            "label": "Queues",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.left.json]: Files d'attente

File: inventory/i18n/fr/menu.inventory.top.json [hunk 6]
```diff
@@ -1,6 +1,29 @@
...
+        "item.product": {
+            "label": "Produits",
+            "help": ""
+        },
+        "item.services.subscription": {
+            "label": "Abonnements",
+            "help": ""
+        },
+        "item.sale.receivables": {
+            "label": "Receivables",
```
Comment (lucaslaurent04) [inline inventory/i18n/fr/menu.inventory.top.json]: Percevables

**Review by lucaslaurent04 — COMMENTED** [review]
I18n

**Comment by ghost** [comment]
> I18n

The mentioned translations have been corrected

**Pull request merged**